### PR TITLE
Add documentation for Existing EKS Cluster Open Source Obsrvability

### PR DESCRIPTION
**Pull request by murubhas**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by elamaran11** [comment]
@murubhas Please fix the Markdown doc issues.

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+This example makes use of CloudWatch, as a metric and log aggregation layer, while X-Ray is used as a trace-aggregation layer. In order to collect the metrics and traces, we use the Open Source ADOT collector. Fluent Bit is used to export the logs to CloudWatch Logs.
+
+In this architecture, AWS X-Ray provides a complete view of requests as they travel through your application and filters visual data across payloads, functions, traces, services, and APIs. X-Ray also allows you to perform analytics, to gain powerful insights about your distributed trace data.
+
+Utilizing CloudWatch and X-Ray as an aggregation layer allows for a fully-managed scalable telemetry backend. In this example we get those benefits while still having the flexibility and rapid development of the Open Source collection tools.
+
+## Objective
+
+This pattern aims to add Observability on top of an existing EKS cluster, with a mixture of AWS native and open source managed AWS services.
```
Comment (elamaran11) [inline docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md:17]: Doc error. This is Open source services only pattern

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# Existing EKS Cluster Open Source Observability Accelerator
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Existing EKS Cluster Open Source Observability pattern, using AWS native tools such as CloudWatch, AWS Managed Prometheus(AMP), AWS Managed Grafana(AMG), and X-Ray and Open Source tools such as Amazon Distro for OpenTelmetry (ADOT) and Prometheus Node Exporter.
+
+![Architecture](../images/CDK_Architecture_diagram.png)
+
+This example makes use of CloudWatch, as a metric and log aggregation layer, while X-Ray is used as a trace-aggregation layer. In order to collect the metrics and traces, we use the Open Source ADOT collector. Fluent Bit is used to export the logs to CloudWatch Logs.
```
Comment (elamaran11) [inline docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md:9]: This is incorrect. Please use docs from Open source pattern,

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+In this architecture, AWS X-Ray provides a complete view of requests as they travel through your application and filters visual data across payloads, functions, traces, services, and APIs. X-Ray also allows you to perform analytics, to gain powerful insights about your distributed trace data.
+
+Utilizing CloudWatch and X-Ray as an aggregation layer allows for a fully-managed scalable telemetry backend. In this example we get those benefits while still having the flexibility and rapid development of the Open Source collection tools.
```
Comment (elamaran11) [inline docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md:13]: Same we are using AMP and AMG for Monitoring. I think copy paste the doc from OSS pattern.

Comment (murubhas) [reply docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md]: Fixed the issues. Please review

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**